azure_sdk/spec/azure_sdk_spec.rb

```diff
@@ -0,0 +1,125 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require_relative 'spec_helper'
+
+describe 'Azure Sdk' do
+  describe 'Management plane' do
+    before(:each) do
+      @tenant_id = '22222222-2222-2222-2222-222222222222'
+      @client_id = '22222222-2222-2222-2222-222222222222'
+      @client_secret = 'VERY_SECRET'
+      @subscription_id = '22222222-2222-2222-2222-222222222222'
+      @active_directory_settings = MsRestAzure::ActiveDirectoryServiceSettings.get_azure_china_settings
+      @minimum_valid_options = {
+          :tenant_id => @tenant_id,
+          :client_id => @client_id,
+          :client_secret => @client_secret,
+          :subscription_id => @subscription_id,
+          :active_directory_settings => @active_directory_settings
+      }
+    end
+
+    after(:each) do
+      Azure::ARM.client.reset!
+    end
+
+    it 'should initialize with client constructor' do
+      arm_client = Azure::ARM.client(@minimum_valid_options)
+      verify_client_defaults(arm_client)
+      verify_management_clients(arm_client)
+    end
+
+    it 'should initialize by defaulting to ENV' do
+      arm_client = Azure::ARM.client({})
```

Comment:
nope. removed

---
In reply to: [76344399](https://github.com/Azure/azure-sdk-for-ruby/pull/433#discussion_r76344399) [](ancestors = 76344399)